queries/models/projeto_subsidio_sppo_encontro_contas/schema.yml

```
@@ -2,190 +2,325 @@ version: 2
+    latest_version: 2
+    versions:
+      - v: 1
+        config:
+          deprecation_date: 2023-12-31
+      - v: 2
+        config:
+          alias: balanco_servico_dia
+    description: "Balanço da receita aferida e esperada por servico e dia para serviços-dia subsidiados. Ver metodologia nas notas técnicas TR/SUBTT Nº 01/2024 e 01/2025"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+            combination_of_columns:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)